src/routes/ViewAllLogs/ViewAllLogs.test.js

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+
+import {
+  fireEvent,
+  render,
+} from '@testing-library/react';
+
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { noop } from 'lodash';
+
+import {
+  buildMutator,
+  Harness,
+} from '@folio/stripes-data-transfer-components/test/helpers';
+
+import '../../../test/jest/__mock__';
+
+import { translationsProperties } from '../../../test/jest/helpers';
+
+import ViewAllLogs from './ViewAllLogs';
```

Comment:
Group the imports, please
```suggestion
import React from 'react';
import {
  fireEvent,
  render,
} from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import { noop } from 'lodash';

import {
  buildMutator,
  Harness,
} from '@folio/stripes-data-transfer-components/test/helpers';

import '../../../test/jest/__mock__';
import { translationsProperties } from '../../../test/jest/helpers';

import ViewAllLogs from './ViewAllLogs';
```